woocommerce-payments.php

```diff
@@ -8,11 +8,11 @@
  * Woo: 5278104:bf3cf30871604e15eec560c962593c1f
  * Text Domain: woocommerce-payments
  * Domain Path: /languages
- * WC requires at least: 5.2
- * WC tested up to: 6.4.1
+ * WC requires at least: 5.4
+ * WC tested up to: 6.5.1
  * Requires at least: 5.7
  * Requires PHP: 7.0
- * Version: 4.1.0
+ * Version: 4.2.0
```

Comment:
Don't change this line! It was supposed to handle by `woorelease` command. PCYsg-z2T-p2#release-the-plugin-with-woorelease

Reply:
OMG. Fixed it.